electron/index.js

```diff
@@ -67,7 +67,7 @@ class CapacitorSplashScreen {
       let capConfigJson = JSON.parse(fs.readFileSync(`./capacitor.config.json`, 'utf-8'));
       this.splashOptions = Object.assign(
         this.splashOptions,
-        capConfigJson.plugins.SplashScreen
+        capConfigJson.plugins.SplashScreen || {}
```

Comment:
This removes the error if the developer has not provided the options in `capacitor.config.json`. This option should be documented better. Very useful.